openedx/openedx-platform
# Fix incorrect course name displayed for CCX courses.

This commit updates the course name for CCX course at Info, Course
and Instructor tabs.

**JIRA tickets**: [OSPR-2557](https://openedx.atlassian.net/browse/OSPR-2557)

**Screenshots**: Attached are some before/after screenshots to show the change

*Note* : The text in the **green box** is the **correct** one whereas the text in the **red box** is the **incorrect** one.

**Course Tab (before)**

<img width="752" alt="Course Tab Before" src="https://user-images.githubusercontent.com/4302268/44440455-c4a45d00-a5e5-11e8-98f4-3c4938577d2d.png">

**Course Tab (after)**

<img width="625" alt="Course Tab After" src="https://user-images.githubusercontent.com/4302268/44440498-f1587480-a5e5-11e8-8b77-3d6294562f78.png">

**Instructor Tab (before)**

<img width="679" alt="Instructor Tab Before" src="https://user-images.githubusercontent.com/4302268/44440551-2b297b00-a5e6-11e8-8193-7cf15e8e47cc.png">

**Instructor Tab (after)**

<img width="716" alt="Instructor Tab After" src="https://user-images.githubusercontent.com/4302268/44440559-35e41000-a5e6-11e8-9ae1-92fb64ed7bf9.png">

**Sandbox URL**: 

LMS: https://pr18808.sandbox.opencraft.hosting/
Studio: https://studio-pr18808.sandbox.opencraft.hosting/

**Testing instructions**:

1. Go to Course Tab ([here](https://pr18808.sandbox.opencraft.hosting/courses/ccx-v1:UniversityOfOpenEducation+UEOEC01+2018_Q4+ccx@1/course/)) and verify that the heading(h2) with class of "hd hd-3 page-title" contains the name of child course
2. Go to Instructor Tab ([here](https://pr18808.sandbox.opencraft.hosting/courses/ccx-v1:UniversityOfOpenEducation+UEOEC01+2018_Q4+ccx@1/instructor#view-course_info)) and verify that the value corresponding to "Course Name" heading in "Basic Course Information" is the name of child course and not parent course.

**Reviewers**
- [ ] @ormsbee 

**Settings**
```yaml
EDXAPP_FEATURES:
  CUSTOM_COURSES_EDX: true
```
